test/regress/regress-2423.txt

```diff
@@ -0,0 +1,3 @@
+;;; TOOL: run-wasm2c
```

Comment:
I tend to prefer to give tests a useful name rather than just regress-XX where possible.   Something like `test/wasm2c/output_no_extension.wast`?

Reply:
But happy to see this land either way.

Reply:
sure